crates/rpc/src/method/get_block_with_txs.rs

```diff
@@ -97,22 +97,22 @@ pub async fn get_block_with_txs(context: RpcContext, input: Input) -> Result<Out
     .context("Joining blocking task")?
 }
 
-impl crate::dto::serialize::SerializeForVersion for Output {
+impl crate::dto::SerializeForVersion for Output {
     fn serialize(
         &self,
-        serializer: crate::dto::serialize::Serializer,
-    ) -> Result<crate::dto::serialize::Ok, crate::dto::serialize::Error> {
+        serializer: crate::dto::Serializer,
+    ) -> Result<crate::dto::Ok, crate::dto::Error> {
         match self {
             Output::Pending {
                 header,
                 transactions,
             } => {
                 let mut serializer = serializer.serialize_struct()?;
-                serializer.flatten(&crate::dto::PendingBlockHeader(header))?;
+                serializer.flatten(header.as_ref())?;
                 serializer.serialize_iter(
                     "transactions",
                     transactions.len(),
-                    &mut transactions.iter().map(crate::dto::TransactionWithHash),
+                    &mut transactions.iter().cloned(),
```

Comment:
Is it possible to drop the clone and add
```
impl crate::dto::SerializeForVersion for &Transaction {
```

Reply:
Good idea 👍